lib/src/screens/exchange/exchange_page.dart

```diff
@@ -469,9 +473,13 @@ class ExchangePage extends BasePage {
     receiveAmountController.addListener(() {
       if (receiveAmountController.text != exchangeViewModel.receiveAmount) {
         _receiveAmountDebounce.run(() {
-          exchangeViewModel.changeReceiveAmount(
-              amount: receiveAmountController.text);
-          exchangeViewModel.isReceiveAmountEntered = true;
+         if (exchangeViewModel.checkIfInputMeetsMinOrMaxCondition(
+                   receiveAmountController.text) ==
+               true) {
+  exchangeViewModel.changeReceiveAmount(
+      amount: receiveAmountController.text);
+  exchangeViewModel.isReceiveAmountEntered = true;
+}
```

Comment:
- format
- duplicate boolean check (i.e bool == true)

Reply:
Removed the double boolean and fixed the formatting